packages/edit-site/src/components/header/document-actions/index.js

```diff
@@ -35,11 +38,10 @@ function useSecondaryText() {
 		};
 	} );
 
-	// Check if current block is a template part:
-	const selectedBlockLabel =
-		selectedBlock?.name === 'core/template-part'
-			? getBlockDisplayText( selectedBlock )
-			: null;
+	// Check if current block is a template area:
+	const selectedBlockLabel = TEMPLATE_AREAS.includes( selectedBlock?.name )
```

Comment:
`non-blocking` - I have been wondering if it might be worth adding a step to `applyFilters` that could be used to extend the list of blocks for the component to show? 🤔

Reply:
Hm, I'm not sure!

I think it definitely makes sense to extend this somehow, but I wonder if filters is how we should do that? (I'm personally not up to date if filters are considered "best practice" for extending JS code anymore)